src/udev/udev-builtin-net_id.c

```diff
@@ -292,12 +294,11 @@ static int dev_pci_slot(struct udev_device *dev, struct netnames *names) {
                 if (i < 1)
                         continue;
 
-                snprintf(str, sizeof str, "%s/%s/address", slots, dent->d_name);
-                if (read_one_line_file(str, &address) >= 0) {
+                if (snprintf_ok(str, sizeof str, "%s/%s/address", slots, dent->d_name) &&
+                    read_one_line_file(str, &address) >= 0)
                         /* match slot address with device by stripping the function */
-                        if (strneq(address, udev_device_get_sysname(names->pcidev), strlen(address)))
+                        if (streq(address, udev_device_get_sysname(names->pcidev)))
```

Comment:
This changes network devices naming in Qemu from `ens3` to `enp0s3` (going from systemd 237 to 238). It seems the new behaviour is correct but was it still intended?

Reply:
This change was reverted in 8eebb6a9e5e74ec0ef40902e2da53d24559b94a4. That commit is in v238-stable btw.